src/crispy.h

```diff
@@ -66,6 +66,7 @@ typedef struct
 	int playercoords;
 	int recoil;
 	int secretmessage;
+	int snowflakes;
```

Comment:
If this isn't a config key anymore, it should get moved down into the "in-game switches and variables" section.